CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -297,10 +291,16 @@ public function upgrade_4_7_15($rev) {
    * @param string $rev
    */
   public function upgrade_4_7_16($rev) {
-    $this->addTask(ts('Upgrade DB to %1: SQL', array(1 => $rev)), 'runSql', $rev);
+    $this->addTask('CRM-19723 - Add icon column to civicrm_option_value', 'addColumn',
+      'civicrm_option_value', 'icon', "varchar(255) COMMENT 'crm-i icon class' DEFAULT NULL");
+    $this->addTask('CRM-19769 - Add color column to civicrm_tag', 'addColumn',
+      'civicrm_tag', 'color', "varchar(255) COMMENT 'Hex color value e.g. #ffffff' DEFAULT NULL");
```

Comment:
Belated nitpick... and this isn't going to block #9629... but `varchar(255)` seems much longer than a color needs. `varchar(8)` is probably enough, and `varchar(32)` would also be pretty comfortable.